pkg/runtime/serializer/protobuf/protobuf.go

```diff
@@ -419,12 +419,6 @@ func (s *RawSerializer) EncodeToStream(obj runtime.Object, w io.Writer, override
 	}
 }
 
-// RecognizesData implements the RecognizingDecoder interface - objects encoded with this serializer
-// have no innate identifying information and so cannot be recognized.
-func (s *RawSerializer) RecognizesData(peek io.Reader) (bool, error) {
-	return false, nil
-}
-
```

Comment:
No longer required to implement the interface - raw serializers cannot
recognize data (no magic bytes to look at)

On Thu, May 5, 2016 at 3:33 AM, Wojciech Tyczynski <notifications@github.com
> wrote:

> In pkg/runtime/serializer/protobuf/protobuf.go
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r62156026>
> :
>
> > @@ -419,12 +419,6 @@ func (s *RawSerializer) EncodeToStream(obj runtime.Object, w io.Writer, override
> >  	}
> >  }
> >
> > -// RecognizesData implements the RecognizingDecoder interface - objects encoded with this serializer
> > -// have no innate identifying information and so cannot be recognized.
> > -func (s *RawSerializer) RecognizesData(peek io.Reader) (bool, error) {
> > -	return false, nil
> > -}
> > -
>
> Don't we need it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e416be61b90b5dc461dc8a84a94d6ecd0d76c216#r62156026>
>